web/html/src/manager/content-management/shared/components/panels/sources/sources.js

```diff
@@ -19,6 +21,7 @@ type SourcesProps = {
 const ModalSourceCreationContent = ({isLoading, softwareSources, onChange}) => {
 
   // TODO: [LuNeves] transform this in an enum and reuse in sources.js as well
+  // TODO: correct browser react warning
```

Comment:
What is this about? Is it something that is caused by this PR? Could it be fixed?

Reply:
Just some react warnings I detected, anyway I removed it and I will open a PR cleaning all the TODOS reminders.